src/js/angular/jdbc/controllers.js

```diff
@@ -32,7 +32,8 @@ function JdbcListCtrl($scope, $repositories, JdbcRestService, toastr, ModalServi
     $scope.getSqlConfigurations = function () {
         // Only do this if there is an active repo that isn't an Ontop or FedX repo.
         // Ontop and FedX repos don't support JDBC.
-        if ($repositories.getActiveRepository()
+        if ($scope.isLicenseValid()
```

Comment:
Inject the license service and use the method from it. Don't rely on the inherited method from the parent scope. Also replace the other similar usages in this file.

Reply:
OK, will fix